docs/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM rust:1-slim
+ARG MDBOOK_VERSION="0.4.5"
```

Comment:
This should come from the makefile to make sure the two are in sync.

Reply:
Are we expecting the quay to run Makefile target to build the image? Also, do I need to add a push or will it be configured within the quay?

Reply:
Quay uses the Dockerfile directly.

Reply:
If quay uses dockerfile directly, then we probably can control the var in Dcokerfile itself @maelk  ?

Reply:
My point was not about the quay building, but the container-image makefile command, that should pass the variable coming from the makefile. Otherwise there is a risk that if they are out of sync, a version x image would be built when the user expects a version y (set in makefile). This could be fixed by using something like --build-arg in the command. 
I don't really expect this command to be used a lot, so I'm also fine leaving things as they are now.

Reply:
True, if only for build local copy of the image, then controlling the var with` --build-arg` make sense. Update it, thanks.